### PR TITLE
Update console and studio urls to app url

### PR DESCRIPTION
**Pull request by defunctzombie**
### Public-Facing Changes
All console.foxglove.dev and studio.foxglove.dev urls now use app.foxglove.dev

### CODE REVIEW

File: docs/1-organization-setup/3-okta-sso.mdx [hunk 1]
```diff
@@ -26,14 +26,14 @@ Optionally, you can enable Okta application tile sign in using these settings:
 
 - **Login initiated by** – Either Okta or App
 - **Login flow** – Redirect to app to initiate login (OIDC Compliant)
-- **Login URI** – Same as the sign-in redirect URL in the previous step (`https://console.foxglove.dev/{YOUR-FOXGLOVE-SLUG}/signin`)
+- **Login URI** – Same as the sign-in redirect URL in the previous step (`https://app.foxglove.dev/~/{YOUR-FOXGLOVE-SLUG}/signin`)
```
Comment (jtbandes) [inline docs/1-organization-setup/3-okta-sso.mdx:29]: this is wrong

File: docs/3-visualization/9-shareable-links.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Build and share deep links with your teammates to open Foxglove with specific la
 To open Foxglove using a specific layout or data source, construct a "deep link" URL using the format:
 
 ```
-https://studio.foxglove.dev/?param1=value2&param2=value2
+https://app.foxglove.dev/~/view?param1=value2&param2=value2
```
Comment (jtbandes) [inline docs/3-visualization/9-shareable-links.md:11]: On this page, which is about constructing links, do we want to explain in words what the `~` is doing?

File: docs/7-recordings.md [hunk 3]
```diff
@@ -5,24 +5,24 @@ description: Recordings represent every data file recorded by your organization'
...
 | **Delete**                 | Delete a data recording imported into your organization                                              | N/A                                                                                                                                                                                       |
-| **Attach contextual data** | Import [MCAP](https://mcap.dev) files with [metadata records](https://mcap.dev/spec#metadata-op0x0c) | N/A                                                                                                                                                                                       |\
+| **Attach contextual data** | Import [MCAP](https://mcap.dev) files with [metadata records](https://mcap.dev/spec#metadata-op0x0c) | N/A                                                                                                                                                                                       | \   |
```
Comment (jtbandes) [inline docs/7-recordings.md:26]: what's going on with this `\`?

File: docs/3-visualization/9-shareable-links.md (hunk 2 above)
Comment (defunctzombie) [reply docs/3-visualization/9-shareable-links.md:11]: I replaced it with `{YOUR-FOXGLOVE-SLUG}`. Thoughts? If not that - what should we say?

Comment (jtbandes) [reply docs/3-visualization/9-shareable-links.md:11]: That seems fine. We could also add a sentence like "You can also use `~` to redirect to whichever organization the user is currently logged into" – or we could just leave it out

File: docs/3-visualization/9-shareable-links.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ To specify your data source, use the `ds` parameter as documented according to y
 For example, a Rosbridge connection may look like the following:
 
 ```
-https://studio.foxglove.dev/?ds=rosbridge-websocket&ds.url=ws%3A%2F%2Flocalhost%3A9090&layoutId=2644147b-f205-456c-ace9-69466aaac0b7
+https://app.foxglove.dev/~/view?ds=rosbridge-websocket&ds.url=ws%3A%2F%2Flocalhost%3A9090&layoutId=2644147b-f205-456c-ace9-69466aaac0b7
```
Comment (jtbandes) [inline docs/3-visualization/9-shareable-links.md:56]: nit: this one doesn't match the new slug placeholder thing you did above. You could keep it if you want to add an explanation of what `~` is, or you could make it match the others

**Review by jtbandes — APPROVED** [review]

File: docs/3-visualization/9-shareable-links.md (hunk 4 above)
Comment (defunctzombie) [inline docs/3-visualization/9-shareable-links.md:56]:
```suggestion
https://app.foxglove.dev/{YOUR-FOXGLOVE-SLUG}/view?ds=rosbridge-websocket&ds.url=ws%3A%2F%2Flocalhost%3A9090&layoutId=2644147b-f205-456c-ace9-69466aaac0b7
```

File: docs/3-visualization/9-shareable-links.md (hunk 2 above)
Comment (defunctzombie) [inline docs/3-visualization/9-shareable-links.md:11]:
```suggestion
https://app.foxglove.dev/{YOUR-FOXGLOVE-SLUG}/view?param1=value2&param2=value2
```

**Pull request merged**